toxcore/net_profile.h

```diff
@@ -0,0 +1,69 @@
+/* SPDX-License-Identifier: GPL-3.0-or-later
+ * Copyright © 2023 The TokTok team.
+ */
+
+/**
+ * Functions for the network profile.
+ */
+#ifndef C_TOXCORE_TOXCORE_NET_PROFILE_H
+#define C_TOXCORE_TOXCORE_NET_PROFILE_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "attributes.h"
+
+/* The max number of packet ID's (should always be >= 256) */
```

Comment:
Should probably always be exactly `UINT8_MAX`, because the protocol demands 1 byte for packet type.